test/integration/dialects/postgres/query-interface.test.js

```diff
@@ -15,6 +17,163 @@ if (dialect.match(/^postgres/)) {
       this.queryInterface = this.sequelize.getQueryInterface();
     });
 
+    describe('createFunction', function () {
+
+      beforeEach( function () {
+        var self = this;
+        //make sure we don't have a pre-existing function called create_job
+        return self.queryInterface.dropFunction('create_job',[{type:'varchar',name:'test'}])
+        //suppress errors here. if create_job doesn't exist thats ok.
+        .catch( function (err) {  });
+      });
+
+      afterEach( function () {
+        var self = this;
+        //cleanup
+        return self.queryInterface.dropFunction('create_job',[{type:'varchar',name:'test'}])
+        //suppress errors here. if create_job doesn't exist thats ok.
+        .catch( function (err) {  });
+      });
+
+      it('creates a stored procedure', function () {
+        var self = this;
+        var body = 'return test;';
+        var options = {};
+
+        //make our call to create a function
+        return new Promise.method( function () {
+          return self.queryInterface.createFunction('create_job', [{type:'varchar',name:'test'}], 'varchar', 'plpgsql', body, options);
```

Comment:
This goes for other cases too where you have wrapped `create / rename / drop Function()` with `Promise.method`, They all return Promise already, no need to wrap them